### PR TITLE
delete saved judge based on judge id

### PR DESCRIPTION
**Pull request by MichelleCCodes**
# Description 
- saves judge to database
- successfully deletes judge
- responds with updated saved_judges after deletion

## Type of Change
- [x] bug fix 
- [x] new feature

## Notes: 
 - There is a bug that occurs after trying to delete the judge. Error is coming from params on frontend. Need to inspect how MUI is updating. Code that introduces bug is commented out for now. The rest of the function works smoothly.

### CODE REVIEW

File: api/profile/profileModel.js [hunk 1]
```diff
@@ -33,21 +33,23 @@ const findById = async (id) => {
   //     cases.push(Object.values(one_case)[0]);
   //   }
   //   book_marked_cases = cases;
```
Comment (katieolson84) [inline api/profile/profileModel.js:35]: Nice work! This looks clean and DRY!

File: api/profile/profileModel.js [hunk 2]
```diff
@@ -33,21 +33,23 @@ const findById = async (id) => {
...
-  //           name: book_marked_judges[i].judge_name,
-  //         })
-  //         .select('*');
-  //       judges.push(Object.values(one_judge)[0]);
-  //     }
-  //     book_marked_judges = judges;
-  //   }
   // }
+
+    if(book_marked_judges.length > 0){
```
Comment (katieolson84) [inline api/profile/profileModel.js:38]: This SonarCloud checked small bugs/syntax and it is recommended some spaces between the curly braces.

File: api/profile/profileRouter.js [hunk 3]
```diff
@@ -415,48 +208,6 @@ router.delete('/pending/:id', authRequired, (req, res) => {
 });
 
 // TODO attach middleware for judge/:name route && case/:id route
```
Comment (katieolson84) [inline api/profile/profileRouter.js:210]: Do we need to leave in these TODO comments?

**Review by katieolson84 — CHANGES_REQUESTED** [review]
Your code changes look good. You removed the @swagger code comments which was a good call since they looked like a reference only. However, you left in some of the 'TODO' comments that went with them. Also, the SonarCloud recommended some spacing adjustments. Once you made the small corrections I will approve the code. Great work!

**Review by paulstgermain — APPROVED** [review]
Good work removing the Swagger code - I'm glad to see such a big chunk of unused code gone. And also good work responding to the requested changes! I see no bugs with this code, it should work great with your work on the front end - approved!

**Review by katieolson84 — APPROVED** [review]
Great job making those small changes- approved!

**Pull request merged**